packages/react-reconciler/README.md

```diff
@@ -0,0 +1,16 @@
+# react
```

Comment:
derp, probably need to do more than simply copy-paste this.

Reply:
😁